### PR TITLE
Hmusum/use remote session repo for finding active session

### PR DESCRIPTION
**Pull request by hmusum**
This is a fix for issues seen where we do e.g. do not restart nodes when there are changes requiring it (we do not compare with previous deployment, since we do not find it).

(First commit is from https://github.com/vespa-engine/vespa/pull/13326)

### CODE REVIEW

File: configserver/src/main/java/com/yahoo/vespa/config/server/session/RemoteSession.java [hunk 1]
```diff
@@ -135,4 +139,34 @@ public void delete() {
...
+        return createSetStatusTransaction(Status.DEACTIVATE);
+    }
+
+    private Transaction createSetStatusTransaction(Status status) {
+        return zooKeeperClient.createWriteStatusTransaction(status);
+    }
+
+    public ApplicationMetaData getMetaData() {
+        return zooKeeperClient.loadApplicationPackage().getMetaData();
+    }
```
Comment (jonmv) [inline configserver/src/main/java/com/yahoo/vespa/config/server/session/RemoteSession.java:169]: Could probably remove most of these from LocalSession, if they're not to be used on that?

**Review by jonmv — COMMENTED** [review]
So the idea is that the remote session is more up to date? Local sessions are set only when deployed locally, while remotes are set whenever deployed? If so, this is absolutely right, and horrible it's been wrong for so long :)

File: configserver/src/main/java/com/yahoo/vespa/config/server/session/RemoteSession.java (hunk 1 above)
Comment (hmusum) [reply configserver/src/main/java/com/yahoo/vespa/config/server/session/RemoteSession.java:169]: I think I removed those that could be removed. I want to do more refactoring in this area, but let's try this first.

**Comment by hmusum** [comment]
Local session for one id exists only on one config server today, so, yes, that's the idea. I think this has been so all the time.

**Pull request merged**